src/lib/pay-request/api_utils/adminUsers.js

```diff
@@ -9,6 +9,16 @@ const adminUsersMethods = function adminUsersMethods(instance) {
     return service
   }
 
+  const findUser = function findUser(email) {
+    const path = '/v1/api/users/find'
+    return axiosInstance.post(path, { username: email })
```

Comment:
This is confusing.
What's the difference between username and email in admin users?
(Looks like username is an email. Does it have separate email property?)

Reply:
This is an excellent for the architects of admin users - it seems to conflate the ideas, they contain exactly the same information but you can only search on the one field.